src/test/java/com/ecore/roles/api/RolesApiTest.java

```diff
@@ -138,7 +128,7 @@ void shouldGetRoleByUserIdAndTeamId() {
 
         getRole(expectedMembership.getUserId(), expectedMembership.getTeamId())
                 .statusCode(200)
-                .body("name", equalTo(expectedMembership.getRole().getName()));
+                .body("name", hasItem(expectedMembership.getRole().getName()));
```

Comment:
The proper assertion method for finding an attribute in a JSON array is hasItem.